blockchain/genesis/genesis.go

```diff
@@ -113,13 +105,18 @@ func defaultConfig() Genesis {
 	}
 }
 
-// TestDefault is the default genesis config for testing
-func TestDefault() Genesis {
+// TestConfig is the genesis config for testing
+func TestConfig() Genesis {
```

Comment:
The function name doesn't explicitly tell that `initTestDefaultConfig` will be called.